README-en.md

```diff
@@ -0,0 +1,29 @@
+# Serenata de amor
+> Fighting corruptions in a massive and effective way
+
+[Jump to the PT-BR version](README.md)*
```

Comment:
I thought github didn't support `🇧🇷` flag:
![image](https://cloud.githubusercontent.com/assets/2338778/17481736/f8c92f4c-5d54-11e6-93bb-ab80e477a674.png)


*OSX or people with emoji installed probably can see the flag, my bad.*